test/integration/api.bats

```diff
@@ -508,12 +576,51 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker version" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	# version
+	run docker_swarm version
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -ge 8 ]
+
+	# verify
+	client_reg='^Client version: [0-9]+\.[0-9]+\.[0-9]+$'
+	server_reg='^Server version: swarm/[0-9]+\.[0-9]+\.[0-9]+$'
+	[[ "${lines[0]}" =~ $client_reg ]]
+	[[ "${lines[5]}" =~ $server_reg ]]
 }
 
-# FIXME
 @test "docker wait" {
-	skip
+	TEMP_FILE=$(mktemp)
+	start_docker 3
+	swarm_manage
+
+	# run after 3 seconds, test_container will exit
+	run docker_swarm run -d --name test_container busybox sleep 3
+	[ "$status" -eq 0 ]
+
+	# make sure container exists and is up
+	run docker_swarm ps -l
+	[ "${#lines[@]}" -eq 2 ]
+	[[ "${lines[1]}" ==  *"test_container"* ]]
+	[[ "${lines[1]}" ==  *"Up"* ]]
+
+	# wait until exist(after 3 seconds)
+	docker_swarm wait test_container > $TEMP_FILE &
+	WAIT_PID=$!
+
+	# after 5 secondes, cat $TEMP_FILE      
+	sleep 5
```

Comment:
Here you could use `retry` as well from #693

maybe something like `retry 10 0.5 cat $TEMP_FILE` (will try 10 times every 0.5 seconds, so 5 seconds total)